tests/unit/helpers.js

```diff
@@ -1354,3 +1369,23 @@ export const fakeSponsoredShelf = Object.freeze({
   impression_data: 'some data',
   impression_url: 'https://mozilla.org/',
 });
+
+export const createInternalAddonWithLang = (addon, lang = 'en-US') => {
```

Comment:
Oh,I see why it is not suffixed with `WithDefaultLang`. I guess that's fine then?